scripts/monitor_chatbot_health.py

```
@@ -0,0 +1,271 @@
+        "ok": True,
+        "bucket": "ok",
+        "status_code": status,
+    }
+
+
+def check_bundle_js(base_url: str, timeout: int) -> dict[str, Any]:
+    """GET /bundle.js and confirm the frontend asset is present."""
+    url = f"{base_url}/bundle.js"
+    try:
+        status, body = _http_get(url, timeout)
+    except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)